### PR TITLE
upload sgfg v6.0.0

### PR DESCRIPTION
**Pull request by zjl88858**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37712)

### CODE REVIEW

File: manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+PackageIdentifier: "dwgltd.sgfg"
+PackageVersion: "6.0.0"
+PackageLocale: "en-US"
+Publisher: "Dawugui LTD"
+PublisherUrl: "https://www.transfer.taipei/"
+PackageName: "神龟翻滚"
```
Comment (Trenly) [inline manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:6]:
```suggestion
PackageName: "SGFG6"
```

File: manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+PackageUrl: "https://getsgfg.z28.web.core.windows.net/"
```
Comment (Trenly) [inline manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:7]:
```suggestion
```
This URL doesn't seem to be valid for me

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: "dwgltd.sgfg"
+PackageVersion: "6.0.0"
+PackageLocale: "en-US"
+Publisher: "Dawugui LTD"
```
Comment (ItzLevvie) [inline manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:4]:
```suggestion
Publisher: "Dawugui Ltd"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:4]: For consistency with Control Panel.

**Review by ItzLevvie — DISMISSED** [review]

**Comment by zjl88858** [comment]
URI: https://www.transfer.taipei/, Validation result: Failed, Http status code: ServiceUnavailable
It seems because of cloudflare js challenge?

File: manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+PackageIdentifier: "dwgltd.sgfg"
+PackageVersion: "6.0.0"
+PackageLocale: "en-US"
+Publisher: "Dawugui Ltd"
+PublisherUrl: "https://www.transfer.taipei/"
```
Comment (Trenly) [inline manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:5]:
```suggestion
# PublisherUrl: "https://www.transfer.taipei/"
```

Comment (Trenly) [reply manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:5]: This should fix the error from cloudflare

Comment (zjl88858) [reply manifests/d/dwgltd/sgfg/6.0.0/dwgltd.sgfg.locale.en-US.yaml:5]: THX.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**